server/src/main/java/com/genymobile/scrcpy/FakeContext.java

```diff
@@ -32,6 +34,7 @@ public String getOpPackageName() {
         return PACKAGE_NAME;
     }
 
+
```

Comment:
Nit: remove unnecessary diff

Reply:
Fixed; somehow an unnecessary newline got in.